website/docs/feature-flag-tutorials/use-cases/a-b-testing.md

```diff
@@ -79,13 +79,12 @@ Using strategy variants in your activation strategy is the canonical way to run
 
 ![This diagram breaks down how strategy variants sit on top activation strategies for flags in Unleash.](/img/tutorial-building-blocks-strategy-variants.png)
 
-A variant has three components that define it:
+A variant has four components that define it:
```

Comment:
it's 3 components: name, weight, and payload. Also the current description matches better, a variant doesn't have a `value`. it's the payload that has a value